src/index.jsx

```diff
@@ -72,17 +75,23 @@ async function init() {
   library.add(faAngleUp, faAngleDown);
 
   render(
-    <div>
+    < div >
```

Comment:
something is off with the linting here, there shouldn't be spaces

Reply:
linting has been fixed